src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1998,18 +2003,53 @@ public override IOperation VisitInvocation(IInvocationOperation operation, int?
             return new InvocationExpression(operation.TargetMethod, visitedInstance, operation.IsVirtual, visitedArguments, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
         }
 
+        private int? _currentInitializerCaptureId = null;
```

Comment:
>private int? _currentInitializerCaptureId = null; [](start = 8, length = 49)

Please keep all fields, which comprise visitor's state, at the beginning of the type declaration